README.md

```diff
@@ -133,15 +135,21 @@ A negation for this specification is also available: `NotEqualIgnoreCase`.
 
 Compares an attribute of an entity with multiple values of a HTTP parameter. E.g. `(..) where gender in (MALE, FEMALE)`.
 
+Supports multiple data types: numbers, booleans, strings, dates, enums.
+
+Usage: `@Spec(path="gender", spec=In.class)`.
+
 HTTP request example:
 
     GET http://myhost/customers?gender=MALE&gender=FEMALE
 
-Supports multiple data types: numbers, booleans, strings, dates, enums.
+or if `paramSeparator` is specified (eg. `@Spec(path="gender", paramSeparator=",", spec=In.class)`):
```

Comment:
as you can see, it's easy to forget regexp, you have "," here, and "\\," above :)